.devcontainer/Dockerfile

```diff
@@ -51,6 +51,11 @@ ARG NEXTEST_VERSION=0.9.14
 RUN curl --proto '=https' --tlsv1.3 -vsSfL "https://github.com/nextest-rs/nextest/releases/download/cargo-nextest-${NEXTEST_VERSION}/cargo-nextest-${NEXTEST_VERSION}-x86_64-unknown-linux-gnu.tar.gz" \
     | tar zvxf - -C /usr/local/bin cargo-nextest
 
+FROM docker.io/golang:${GO_VERSION}-bullseye as actionlint
+COPY bin/scurl /usr/local/bin/scurl
+RUN scurl -v https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash \
```

Comment:
can we pin this to a release or tag or sha?  worried about main changing out from under us.

Reply:
Yup! Sounds good.

I initially pinned this to the latest release (`v1.6.14`). It seems that the release has an issue when instantiating a nested env context with a parent value, e.g

```
env: 
   LINKERD_REGISTRY: ghcr.io

jobs:
  steps
  - name: Linkerd test
     env:
        LINKERD_REGISTRY: ${{ env.LINKERD_REGISTRY }}
```

ref: https://github.com/rhysd/actionlint/issues/158

v1.6.13 should work well for now.

Reply:
what about pinning to a release or tag or sha of the download script?  right now we're downloading the script from a floating main

Reply:
I think we need 1.6.15 now... https://github.com/rhysd/actionlint/issues/158#issuecomment-1168598736